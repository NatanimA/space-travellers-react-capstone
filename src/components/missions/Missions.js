import React from 'react';
import MissionsList from './MissionsList';

const Missions = () => (
  <section className="section-containers section-mission">
    <div>
      <MissionsList />
    </div>
  </section>
);

export default Missions;
