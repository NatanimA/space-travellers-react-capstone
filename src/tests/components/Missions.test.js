import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../../redux/configureStore';
import Missions from '../../components/missions/Missions';

test('renders rockets component snapshot', () => {
  const { asFragment } = render(
    <Provider store={store}>
      <BrowserRouter>
        <Missions />
      </BrowserRouter>
    </Provider>,
  );
  expect(asFragment()).toMatchSnapshot();
});
