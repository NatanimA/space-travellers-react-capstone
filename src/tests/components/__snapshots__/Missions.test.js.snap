// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders rockets component snapshot 1`] = `
<DocumentFragment>
  <section
    class="section-containers section-mission"
  >
    <div>
      <table
        class="table table-striped table-bordered table-hover"
      >
        <thead>
          <tr>
            <th>
              Missions
            </th>
            <th>
              Description
            </th>
            <th>
              Status
            </th>
            <th>
               
            </th>
          </tr>
        </thead>
        <tbody />
      </table>
    </div>
  </section>
</DocumentFragment>
`;
